workflow/rules/post-analysis.smk

```diff
@@ -103,14 +112,16 @@ rule bedGraphToBigWig:
 
 rule create_igv_bigwig:
     input:
-        "resources/ref/genome.bed",
-        expand("results/big_wig/{sample}.bigWig", sample=samples.index)
+        genome="resources/ref/genome.bed",
```

Comment:
Is this input actually needed? From what I see, it is not used in `params:` or `shell:` below, right?

Reply:
Thanks, that's right, the genome input variable is not necessary here.